features/steps/get_task_steps.py

```diff
@@ -8,4 +8,23 @@
 # handle to the task manager application
 from taskmanager.app.task_manager import TaskManager
 
-# TODO: GRUPPE 3 - Add BDD steps for getting a specific task by name
\ No newline at end of file
+# TODO: GRUPPE 3 - Add BDD steps for getting a specific task by name
```

Comment:
TODO bitte löschen, sonst taucht das noch in der TODO Liste auf, obwohl adressiert.